python/cuxfilter/charts/datashader/plots.py

```diff
@@ -680,7 +684,6 @@ def generate_chart(self):
             y_range=self.y_range,
             width=self.width,
             height=self.height,
-            output_backend="webgl",
```

Comment:
just curious the motivation to remove this?

Reply:
So initially there was an automatic switch in rendering the points, nodes<=10_000 used bokeh rendering, which is where the output_backend="webgl" was used, but there was a lot of difference between the datashader based point rendering and bokeh based point/graph rendering, one of the things being circle_radius scales. To avoid confusion, I thought its best to have bokeh based graph rendering as a separate chart(will have to add it in 0.16)